lib/whitehall/search_index.rb

```diff
@@ -18,7 +28,7 @@ def self.indexer_class
     end
 
     def self.rummager_host
-      Plek.find('search')
+      Plek.find('rummager')
     end
 
     def self.add(instance)
```

Comment:
Yep makes sense. `self.for` is a bit weird and `self.add` is the more normal code path I think.